table/table-tests.ts

```diff
@@ -0,0 +1,15 @@
+import * as table from 'table';
+
+const border = table.getBorderCharacters('norc');
+
+const data = [
+  ['first name', 'last name'],
+  ['jane', 'doe'],
+  ['john', 'doe']
+];
+
+const config = {
```

Comment:
Unfortunately, TypeScript won't typecheck misspelled parameter names if it appears like this. (Try it out: TS will infer e.g. ` { brdr: JoinStruct }`, which technically implements `TableUserConfig` since all properties on that interface are optional.) Either do `const config: table.TableUserConfig` or inline it.